packages/types/src/events/shared-channel.ts

```diff
@@ -0,0 +1,135 @@
+interface SharedChannelTeamItem {
```

Comment:
Not sure, but should all of these be exported?

Reply:
Nope! As soon as we do, we can't change them or rename them without making a breaking change. Anything not exported is only used within the file it exists in. In this case, all of these unexported interfaces are only re-used within the `*Event` payloads present in this file.